lib/util/support.js

```diff
@@ -24,12 +24,24 @@ function isGetUserMediaSupported() {
     || !!(navigator.mozGetUserMedia);
 }
 
+/**
+ * @returns {boolean} - true if the browser is explicitly unsupported.
+ *  Note: this function returning false does not mean that browser is supported.
+ */
+function isExplicitlyUnsupportedBrowser() {
+  if (typeof navigator !== 'undefined' && typeof navigator.userAgent === 'string') {
+    return /Edge/.test(navigator.userAgent);
```

Comment:
@manjeshbhargav - Looks like your [fix](https://github.com/twilio/twilio-video.js/pull/756) to handle case for Edg(e) did not make it here? Do you know why ?

Reply:
@makarandp0 Good catch! I just merged from the `support-1.x-chrome-planb` branch. I forgot to look at the changes in `master`.